src/Conduit/PipelineManager.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using System;
+
+namespace Conduit
+{
+    internal sealed class PipelineManager<T> : IPipelineManager<T>
+    {
+        private readonly IOptionsMonitor<PipelineOptions<T>> _options;
+        private readonly IServiceScopeFactory _serviceScopeFactory;
+
+        public PipelineManager(IOptionsMonitor<PipelineOptions<T>> options, IServiceScopeFactory serviceScopeFactory)
+        {
+            _options = options ?? throw new ArgumentNullException(nameof(options));
+            _serviceScopeFactory = serviceScopeFactory ?? throw new ArgumentNullException(nameof(serviceScopeFactory));
+        }
+
+        public IPipeline<T> Get(string name)
+        {
+            if (string.IsNullOrWhiteSpace(name))
+            {
+                name = "_default";
```

Comment:
Isn't the default pipeline called Constants.DefaultPipeline?
```suggestion
                name = Constants.DefaultPipeline;
```